src/Lecture1.hs

```diff
@@ -77,7 +80,11 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z =
+  let maxNum a b c = max a (max b c)
+      minNum a b c = min a (min b c)
+   in maxNum x y z - minNum x y z
```

Comment:
You don't need to define `maxNum` and `minNum` as functions as the function arguments are already in scope for them
```suggestion
  let maxNum = max x (max y z)
      minNum = min x (min y z)
   in maxNum - minNum
```